composer.json

```diff
@@ -12,15 +12,15 @@
     "require": {
         "php": "^7.1|^8.0",
         "facade/ignition-contracts": "~1.0",
-        "illuminate/pipeline": "^5.5|^6.0|^7.0|^8.0",
+        "illuminate/pipeline": "^5.5|^6.0|^7.0|^8.0|^9.0",
         "symfony/http-foundation": "^3.3|^4.1|^5.0",
         "symfony/mime": "^3.4|^4.0|^5.1",
         "symfony/var-dumper": "^3.4|^4.0|^5.0"
```

Comment:
This needs to be updated to support version ^6.0 of symfony/var-dumper as Laravel 9 requires this version.